media/CfgEditor/cfgeditor.js

```diff
@@ -0,0 +1,325 @@
+
+// Main function that gets executed once the webview DOM loads
+function main() {
+  const inputPathCodicon = document.getElementById('inputPathCodicon');
+  inputPathCodicon.addEventListener('click', clickInputPathCodicon);
+
+  const importEnabled = document.getElementById('importEnabled');
+  const importDisabled = document.getElementById('importDisabled');
+  const importLink = document.getElementById('linkEnableImport');
+  importEnabled.addEventListener('click', disableImportStep);
+  importDisabled.addEventListener('click', enableImportStep);
+  importLink.addEventListener('click', enableImportStep);
+
+  const optimizeEnabled = document.getElementById('optimizeEnabled');
+  const optimizeDisabled = document.getElementById('optimizeDisabled');
+  const optimizeLink = document.getElementById('linkEnableOptimize');
+  optimizeEnabled.addEventListener('click', disableOptimizeStep);
+  optimizeDisabled.addEventListener('click', enableOptimizeStep);
+  optimizeLink.addEventListener('click', enableOptimizeStep);
+
+  const quantizeEnabled = document.getElementById('quantizeEnabled');
+  const quantizeDisabled = document.getElementById('quantizeDisabled');
+  const quantizeLink = document.getElementById('linkEnableQuantize');
+  quantizeEnabled.addEventListener('click', disableQuantizeStep);
+  quantizeDisabled.addEventListener('click', enableQuantizeStep);
+  quantizeLink.addEventListener('click', enableQuantizeStep);
+
+  const codegenEnabled = document.getElementById('codegenEnabled');
+  const codegenDisabled = document.getElementById('codegenDisabled');
+  const codegenLink = document.getElementById('linkEnableCodegen');
+  codegenEnabled.addEventListener('click', disableCodegenStep);
+  codegenDisabled.addEventListener('click', enableCodegenStep);
+  codegenLink.addEventListener('click', enableCodegenStep);
+
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const profileLink = document.getElementById('linkEnableProfile');
+  profileEnabled.addEventListener('click', disableProfileStep);
+  profileDisabled.addEventListener('click', enableProfileStep);
+  profileLink.addEventListener('click', enableProfileStep);
+
+  const outputPathCodicon = document.getElementById('outputPathCodicon');
+  outputPathCodicon.addEventListener('click', clickOutputPathCodicon);
+
+  const inputModelSearchCodicon = document.getElementById('inputModelSearch')
+  inputModelSearchCodicon.addEventListener('click', inputModelSearchClick);
+
+  const outputPathSearchCodicon = document.getElementById('outputPathSearch');
+  outputPathSearchCodicon.addEventListener('click', outputPathSearchClick);
+
+  const modelTypeRadio = document.getElementById('modelTypeRadio');
+  modelTypeRadio.addEventListener('click', modelTypeClick);
+
+  const foldButton = document.getElementById('foldAdvancedOptions');
+  foldButton.addEventListener('click', clickFoldAdvencedOptions);
```

Comment:
```suggestion
  foldButton.addEventListener('click', clickFoldAdvancedOptions);
```

There are few more copy and paste errors :sweat_smile: